Button markup

hein-obox opened the pull request:
## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

heinvv committed:
Sanitize Url

hein-obox commented:
Comment from Myk: 
<img width="624" alt="Screenshot 2024-07-21 at 17 01 04" src="https://github.com/user-attachments/assets/6f28e9c3-5d27-4ea8-8d51-d46e6a15a216">

heinvv committed:
PR Review

louiswol94 commented on assets/dev/js/editor/utils/helpers.js:
```diff
@@ -2,6 +2,7 @@ import ColorPicker from './color-picker';
 import DocumentHelper from 'elementor-editor/document/helper-bc';
 import ContainerHelper from 'elementor-editor-utils/container-helper';
 import DOMPurify from 'dompurify';
+import { isValidAttribute } from 'dompurify';
```
```suggestion
import DOMPurify, { isValidAttribute } from 'dompurify';
```

hein-obox replied:
Updated

heinvv committed:
PR Review

KingYes committed:
Internal: Added PLG for IO [ED-15238] (#28145)

KingYes committed:
Internal: Fix JS lint [ED-15238] (#28150)

ronenelementor committed:
Tweak: Dynamic Tags - Advanced - Fallback setting fix [ED-15390] (#28148)

Co-authored-by: ElementorBot <48412871+elementorbot@users.noreply.github.com>

nicoladj77 committed:
Fix: Floating Buttons - Messenger link - main (#28103) [ED-15365] (#28120)

baghdasarovelementor committed:
Internal: Add a Dependency Injection Framework to Elementor [ED-14953] (#27841)

heinvv committed:
Error handling

hein-obox committed:
Merge branch 'main' into internal/widget-button-markup

mykytamurzin committed:
Internal: Fix and unskip flaky Nested tab test (#28157)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

elementorbot committed:
Merge branch 'main' into internal/widget-button-markup

mserino committed:
Fix: Floating Buttons v9 - fix transform when RTL [ED-15400] (#28159)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [ ] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

elementorbot committed:
Merge branch 'main' into internal/widget-button-markup
